pkg/agent/run.go

```diff
@@ -170,7 +175,18 @@ func gatherAndOutputData(ctx context.Context, config Config, preflightClient *cl
 		}
 		log.Println("Data saved locally to", OutputPath)
 	} else {
-		postData(config, preflightClient, readings)
+		//TODO Change backoff parameters to those desired
+		backOff := backoff.NewExponentialBackOff()
+		backOff.MaxElapsedTime = time.Duration(BackoffMaxTime) * time.Second
```

Comment:
I don't think you need `time.Duration()` here, I think that you can just multiply `time.Second` by the number of seconds.

Reply:
I tried it as an integer and Go didn't like it. I think your other comment is the way to go with initialising as `time.Duration()` is the way to go